pkg/webservice/handlers.go

```diff
@@ -291,3 +293,33 @@ func getContainerHistory(w http.ResponseWriter, r *http.Request) {
 	}
 
 }
+
+func getClusterConfig(w http.ResponseWriter, r *http.Request) {
+	writeHeaders(w)
+	jsonOutput := false
```

Comment:
Normally if an HTTP client expects a certain format of the response from the server, an HTTP header such as "Accept: application/json" needs to be set. Instead of adding a query parameter to do so. This is referring to doc like https://restfulapi.net/content-negotiation/.

Reply:
changed, default is yaml